source/upgrade.txt

```diff
@@ -99,6 +99,14 @@ Procedure
 
 #. Launch the fully-featured edition of |compass|.
 
+.. note::
```

Comment:
So, as is we have this note tucked in the "migration" section of this page. This note doesn't really apply to the migration procedure. Instead, I'd suggest moving this to the bottom of the `Manually Install the Latest Version of Compass` section, since if they are manually installing that's when they are most likely to run into this issue.